spec/factories/favorites.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :favorite do
+    user_id 1
+tweet_id 1
+  end
+
+end
```

Comment:
Formatting is always off on new `FactoryGirl` factories. When you get around to it, make sure you clean the spacing and what not off.